src/utp_stream.cpp

```diff
@@ -555,6 +571,9 @@ struct utp_socket_impl
 	// average RTT
 	sliding_average<16> m_rtt;
 
+    // lowest RTT observed during the whole existence time of connection
+    boost::int32_t m_min_rtt;
```

Comment:
this is signed. when you initialize it in the constructor you set it to the *unsigned* max. they should be consistent (and I think signed may be slightly less error prone)